src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int bottomBase;
+    private int topBase;
+    private int side;
+
+    public IsoscelesTrapezoid(String color, String name, int bottomBase, int topBase, int side) {
+        super(color, name);
+        this.bottomBase = bottomBase;
+        this.topBase = topBase;
+        this.side = side;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (((bottomBase + topBase) / 2)
+                * (Math.sqrt((Math.pow(side, 2))
+                - Math.pow((((Math.pow((bottomBase - topBase), 2))
+                + (Math.pow(side, 2)) - (Math.pow(side, 2)))
+                / (2 * (bottomBase - topBase))), 2))));
```

Comment:
The `calculateArea` method uses integer division, which can lead to precision loss. Consider using `double` for calculations to ensure accuracy, especially when dealing with square roots and divisions.